src/components/choropleth.js

```diff
@@ -149,6 +149,7 @@ function ChoroplethMap({
         .on('click', handleClick)
         .attr('pointer-events', 'none');
 
+      regionSelection.select('title').remove();
```

Comment:
Hey, could you check your PR again? The percentage fix doesn't seem to be here.

Let me know once you've resolved that :)

Reply:
@jeremyphilemon 
The percentage calculation was never a problem. The calculation is happening correctly. 
Every time user clicks on the different graph option the  "title" is getting appended with old titles. 

![image](https://user-images.githubusercontent.com/43884208/80937540-2e34c100-8df3-11ea-9f4e-59f498c51a71.png)

As the topmost title is the oldest one, on hover it shows that. 

So, the fix to this issue is to remove the old title and add a new one.